vhdbuilder/packer/cleanup.sh

```diff
@@ -38,3 +38,30 @@ if [[ "$MODE" != "default" ]]; then
     az image delete -n ${IMAGE_NAME} -g ${AZURE_RESOURCE_GROUP_NAME}
   fi
 fi
+
+#cleanup imported sig image version
+if [[ -n "${IMPORTED_IMAGE}" ]]; then
+  id=$(az sig image-version show -e 1.0.0 -i ${IMPORTED_IMAGE} -r ${SIG_GALLERY_NAME} -g ${AZURE_RESOURCE_GROUP_NAME} | jq .id)
+  if [ -n "$id" ]; then
+    echo "Deleting sig image-version 1.0.0 ${IMPORTED_IMAGE} from gallery ${SIG_GALLERY_NAME} rg ${AZURE_RESOURCE_GROUP_NAME}"
+    az sig image-version delete -e 1.0.0 -i ${IMPORTED_IMAGE} -r ${SIG_GALLERY_NAME} -g ${AZURE_RESOURCE_GROUP_NAME}
+  fi
+fi
```

Comment:
is this needed? Removing the imported image definition should remove all the image versions, right?

Reply:
The image-definition throws an error if I try to delete it without deleting the nested resources. If I delete the image-version first it works fine.